docs/user-guide/configure-apiml-zosmf-workflow.md

```diff
@@ -0,0 +1,113 @@
+# Configuring API ML with z/OSMF Workflows
+
+After you install Zowe, you can register and execute the z/OSMF workflows in the web interface to perform a range of
+API ML/Zowe configuration tasks. z/OSMF helps to simplify the API ML configuration tasks and does not require the level of
+expertise that is needed to perform manual API ML/Zowe configuration. This configuration method also runs the `zwe install` and `zwe init`
+command to initialize Zowe z/OS runtime.
+
+:::info Required role: system programmer
+:::
+
+Ensure that you meet the following requirements before you start your Zowe configuration:
+
+- Install and configure z/OSMF
+- Install Zowe with an SMP/E build, PSWI, or a convenience build
+
+You can complete the following tasks with the **Configuration of API ML/Zowe 3.0** workflow:
```

Comment:
Is this the correct name of the workflow? Configuration of API ML/Zowe 3.0

Reply:
right, change name to: Stand-alone Zowe API ML Configuration

Reply:
updated